app/assets/javascripts/home.coffee

```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
```

Comment:
we can remove this -- I'm guessing if / when we add some javascript, we're going to want to use something other than coffeescript (although, I have a soft spot for coffee script and think its underrated 😄 )